app/views/projects/show.html.erb

```diff
@@ -1,44 +1,20 @@
-<div class="container mx-auto">
-  <div class="mt-5">
-    <div class="mb-3">
-      <%= @project.title %>
-    </div>
-  </div>
-
-  <div>
-    <h4>Maintainers</h4>
-    <%= link_to 'Add Maintainer', edit_project_maintainers_path(@project), class: 'btn btn-primary' %>
-      <ul class="list-group list-group-flush">
-        <% @project.maintainers.each do |maintainer| %>
-          <li class="list-group-item"><%= link_to maintainer.first_name, maintainer, class: 'nav-link' %></li>
+<%= render 'projects/header', project: @project %>
+<div class="page-body">
+  <div class="container-xl">
+    <div class="row row-cards">
+      <div class="col-sm-6 col-lg-3">
+        <%= render 'projects/counter_overview', icon_path: 'static/@tabler/icons/icons/key.svg' do %>
+          <%= link_to "#{@project.accounts.count} Accounts", project_accounts_path(@project) %>
         <% end %>
-      </ul>
-  </div>
-
-  <div>
-    <h4>Accounts</h4>
-    <% if policy(@project).edit? %>
-      <%= link_to 'Add new Account', new_project_account_path(@project), class: 'btn btn-primary' %>
-    <% end %>
-    <% if @project.accounts.any? %>
-      <%= link_to 'Accounts', project_accounts_path(@project), class: 'btn btn-primary' %>
-    <% else %>
-      <p>No accounts could be found for this project.</p>
-    <% end %>
-  </div>
-  <div>
-    <h4>Links</h4>
-    <% if policy(@project).edit? %>
-      <%= link_to 'Add new Link', new_project_link_path(@project), class: 'btn btn-primary' %>
-    <% end %>
-    <% if @project.links.any? %>
-      <%= link_to 'Links', project_links_path(@project), class: 'btn btn-primary' %>
-    <% else %>
-      <p>No links could be found for this project.</p>
-    <% end %>
+      </div>
+      <div class="col-sm-6 col-lg-3">
+        <%= render 'projects/counter_overview', icon_path: 'static/@tabler/icons/icons/users.svg' do %>
+          <%= link_to "#{@project.maintainers.count} Maintainers", edit_project_maintainers_path(@project) %>
```

Comment:
A saját projektjeimen at írja, hogy "1 Maintainers", meg tudjuk itt it oldani a többesszám kezelését?